gigareview/console/tests/functional/EMReportJobCest.php

```diff
@@ -48,12 +61,17 @@ public function tryToSeeNoResultsManuscriptReportNotStoredToTable(FunctionalTest
         file_put_contents($noResultCsvReportDir.$tempNoResultCsvReportName, "No Results");
 
         $I->runShellCommand("./yii_test fetch-reports/fetch", false);
+        $I->canSeeInDatabase("ingest", ["file_name"=>$tempNoResultCsvReportName, "report_type"=>1, "fetch_status"=>3, "parse_status"=>null, "store_status"=>null, "remote_file_status"=>null]);
+        $I->cantSeeInDatabase("ingest", ["file_name"=>$tempNoResultCsvReportName, "report_type"=>1, "fetch_status"=>3, "parse_status"=>0, "store_status"=>0, "remote_file_status"=>0]);
+
         $I->runShellCommand("/usr/local/bin/php /app/yii_test manuscripts-q/run --verbose", false);
+        $I->canSeeInDatabase("ingest", ["file_name"=>$tempNoResultCsvReportName, "report_type"=>1, "fetch_status"=>3, "parse_status"=>null, "store_status"=>null, "remote_file_status"=>null]);
+        $I->canSeeInDatabase("ingest", ["file_name"=>$tempNoResultCsvReportName, "report_type"=>1, "fetch_status"=>3, "parse_status"=>0, "store_status"=>0, "remote_file_status"=>0]);
 
         unlink($noResultCsvReportDir.$tempNoResultCsvReportName);
```

Comment:
If there is failure in any of the test before that line, the ``unlink`` method won't be called.
Better conditionally calling ``unlink`` in an ``_after()`` function.